framework/bundles/org.jboss.tools.rsp.server/src/main/java/org/jboss/tools/rsp/server/model/internal/publishing/ServerPublishStateModel.java

```diff
@@ -345,4 +352,74 @@ private DeployableReference cloneReference(DeployableReference ref) {
 		return ref == null ? null : new DeployableReference(ref.getLabel(), ref.getPath());
 	}
 
+	private void kickAutoPublisher() {
```

Comment:
initially thought - by reading the name - that this would start an autopublish thread that then would publish regularly. Reading the code, then found out that this was a one-shot publishing. I'd thus suggest using a more speaking name? ex. #publishAutoPublisher?